csv_generator/consumers/utils.py

```diff
@@ -0,0 +1,59 @@
+import pendulum
+
+
+MS_TYPES = {
+    'Research Article': 'RA',
+    'Editorial': 'ED',
+    'Feature': 'POV',
+    'Initial Submission Research Article': 'IS/ISRA',
```

Comment:
Clarifying whether we need the mapping. If we do, the `/` was noting where we changed the code (from previously `IS` now `ISRA`)

Reply:
Ok just let me know 👍

Reply:
Okay, it doesn't seem that we need the mapping now. But in the future we will probably need a mapping for renamed types. But that we can think about it then.

Reply:
Ok I will remove the mapping and just return the original type.